confconsole.py

```diff
@@ -330,32 +357,37 @@ def usage(self):
             self.console.msgbox("Error", error)
             return "networking"
 
-        #tklbam integration
-        try:
-            tklbam_status = executil.getoutput("tklbam-status --short")
-        except executil.ExecError, e:
-            if e.exitcode in (10, 11): #not initialized, no backups
-                tklbam_status = e.output
-            else:
-                tklbam_status = ''
+        # tklbam integration
+        tklbamstatus_cmd = subprocess.run(
+                ['which', 'tklbam-status'], stdout=PIPE).stdout
+        if tklbamstatus_cmd:
+            tklbam_status = subprocess.run(
+                    [tklbamstatus_cmd, "--short"], stdout=PIPE).stdout
+        else:
+            tklbam_status = (b"TKLBAM not found - please check that it's "
+                             b"installed.")
 
-        #display usage
+        tklbam_status = tklbam_status.decode('utf-8')
```

Comment:
Small critique of style

`subprocess.run` has an argument `encoding=<encoding>`, which can be used to make subprocess.run return a string. Which would allow us to use string literals as opposed to byte literals (and in turn reduce the number of decodes)

Reply:
Ahh, I should have perhaps read all of your reviews before applying the fixes, as this would also apply to inithooks... Anyway.

Reply:
Actually after looking at this a little closer, AFAIK you still need to use `.decode()` to turn a binary string into a "normal" string. Although rather than hardcoding `utf-8` we probably should set that somewhere and use that value, so then changing it to some other encoding isn't too hard...

Reply:
Scratch my last post... After a closer look I agree that the code could be better written using `encoding=` in the `subprocess.run()` call. So that's what I've done...